integration_tests/README.md

```diff
@@ -4,7 +4,11 @@
 
 - Install `Node.js`
 
-- Open terminal in the `tari-project\integration_tests` folder and run `npm install`
+- Open terminal in the `tari-project\integration_tests` folder and run 
+  ```
+  npm install
+  npm install multimap --save
```

Comment:
This seems wrong...

Reply:
yeah if multimap is a new dependency it should be added to `package.json`

Reply:
Correct, I had to do it once only and then on another system it was not required, due to its inclusion in the package file, will remove.